### PR TITLE
Readability improvements & fixes.

### PR DESCRIPTION
**Pull request by jackbondpreston**
```
Fix some incorrect numbering in the Linux setup guide and improve
readability in some places.
```

### CODE REVIEW

**Review by RileyEv — APPROVED** [review]

**Review by ChrisGora — DISMISSED** [review]
Let's change all files to say that the Go version needs to be "1.13 or higher". 
Otherwise looks good 👍

File: go-install/lab-machines.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ export PATH="$PATH:$GOPATH/bin"
 
 3. Save the file and exit.
 
-4. Close and open any terminal windows. (If you are working over **ssh** you can use the command `source ~/.bashrc`)
+4. Close and re-open any terminal windows. If you are working over `ssh` you can use the command `source ~/.bashrc`.
 
 5. Verify your installation with the command `go version`. The version **must be `1.13`**.
```
Comment (RileyEv) [inline go-install/lab-machines.md:19]:
```suggestion
5. Verify your installation with the command `go version`. The version **will be `1.13`**.
```

File: go-install/linux.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 ```
 
-4. Save the file and exit.
+1. Save the file and exit.
 
-5. Close and open any terminal windows.
+2. Close and re-open any terminal windows.
 
-6. Verify your installation with the command `go version`. The version should be 1.14.3
+3. Verify your installation with the command `go version`. The version should be `1.14.3`.
```
Comment (RileyEv) [inline go-install/linux.md:21]:
```suggestion
3. Verify your installation with the command `go version`. The version **will be `1.15.1`**.
```

File: go-install/linux.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
 # Setup instructions for Linux
 
-2. Run the following commands in terminal
+1. Run the following commands in terminal:
 
 ```
 wget https://dl.google.com/go/go1.14.3.linux-amd64.tar.gz
 sudo tar -C /usr/local -xzf go1.14.3.linux-amd64.tar.gz
```
Comment (RileyEv) [inline go-install/linux.md:7]:
```suggestion
wget https://dl.google.com/go/go1.15.1.linux-amd64.tar.gz
sudo tar -C /usr/local -xzf go1.15.1.linux-amd64.tar.gz
```

**Review by RileyEv — CHANGES_REQUESTED** [review]

**Review by RileyEv — APPROVED** [review]

**Pull request merged**